perf/tools/aggregate.lua

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env tarantool
+
+-- File to aggregate the benchmark results from JSON files to the
+-- format parsable by the InfluxDB line protocol [1]:
```

Comment:
The tool is still working, but it prints the difference in RPS instead of seconds (due to the output change). I suppose that this is OK, since we can still analyze the difference.
```sh
LUA_PATH="perf/lua/?.lua;;" src/tarantool perf/tools/compare.lua /tmp/t1 /tmp/t1 /tmp/t2
       /tmp/t1          /tmp/t1          /tmp/t2
get_0  3100261  3100261 (+  0%)  2958616 (-  4%)
```
I've moved it to the tools directory to avoid confusion.

Reply:
Okay, thanks!